src/redesign/components/HomeBlogPreview.jsx

```diff
@@ -323,14 +324,25 @@ function BlogTags({ tags }) {
   );
 }
 
+const handleImageLoad = (path) => {
+  // Try to load the image
+  try {
+    return require("../images/posts/" + path);
```

Comment:
```suggestion
    return require("../../images/posts/" + path);
```

Reply:
We're gradually trying to deprecate the `/redesign` folder, including its images, and the whole thing will be fully removed in an upcoming fix, but for now, this should reference the standard images folder, not the one inside `/redesign`